manager/catalog_manager.go

```diff
@@ -103,12 +103,18 @@ func SetEnv() {
 					}
 					newCatalog := Catalog{}
 					newCatalog.CatalogID = tokens[0]
-					newCatalog.URL = tokens[1]
+					url := tokens[1]
+					index := strings.Index(tokens[1], "://")
```

Comment:
using `url` instead `tokens[1]` here (and below) would be more clear.  Otherwise lgtm